.github/PULL_REQUEST_TEMPLATE.md

```
@@ -13,31 +13,9 @@ What is changing and why? If someone is reading this 2 years from now to track d
-  - [ ] [All interesting/clever code has a comment adding context](https://stackoverflow.blog/2021/12/23/best-practices-for-writing-code-comments/)
-- [ ] Module level documentation is complete
-  - [ ] `README.md`
-  - [ ] `__manifest__.py`
-- [ ] PR is being merged into the correct target branch (e.g. dev#1234 into E2E)
-- [ ] Mindfully select reviewer(s)
-    
-  - Spread the love!  It's easy to request the same person each time, but that keeps us from getting better
-  - You can add multiple reviewers. Don't be shy
-  - If the merge is large, contains a lot of other developers code, feels complicated, etc... Please add your pod's Development SME as a reviewer
-
 ###  **EXTREMELY IMPORTANT**
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)